lib/aca_entities/payment_transactions/contracts/transaction_contract.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Validates the presence of params and type of effective date.
+module AcaEntities
+  # Payment Transaction
+  module PaymentTransactions
```

Comment:
Can you change the naming to AcaEntities::Contracts::PaymentTransactionContract?

Reply:
Removed payment transactions namespace.